docs/how-to/namespace-transition.md

```diff
@@ -16,28 +16,57 @@ To avoid these issue we recommend taking the proactive approach described in
 [Recommended Migration](#recommended-migration) to ensure continued good
 functionality of your installed events system.
 
-**Note that the current instructions only cover installation based on the
-release from the `knative-gcp` repository. We will update the page with specific
-`gcloud` instructions as the release becomes available there as well.**
+**Note that the [Recommended Migration](#recommended-migration) instructions
+below cover installation based on the release from the `knative-gcp` repository.
+[Recommended Migration for CloudRun](#recommended-migration-for-cloudrun)
+contains instructions for the upgrade when using the
+[Events for Cloud Run for Anthos](https://cloud.google.com/kuberun/docs/events/anthos/quickstart)
+offering.**
 
 ## Recommended Migration
 
+Migrating your events resources to the new namespace involves the following
+steps:
+
+1. Setting up authentication for the new namespace.
+1. Upgrading to the new version of events.
+1. Verifying the correct functionality of already deployed resources.
+
+### Setup Authentication
+
+We assume here that the desire is to keep the existing authentication mechanism
+and provide the steps to correct the permissions to work in the new namespace.
+Next follow the upgrade instructions for authentication with either Kubernetes
+Secrets or Workload Identity, depending on what was used in the old namespace.
+After you've completed the upgrade verify that everything is working correctly
+by following the steps in the last section.
+
+#### Using Kubernetes Secrets
+
 1. Run the following commands before installing the new `v0.20` release. We
```

Comment:
In GCP and K8s docs, individual shell commands are normally separated by additional steps that explain the desired state and give commands for verifying that state, so users are confident they know when to proceed and when to bail out of the process. In this case, since we are doing whole-cluster surgery, I think a bit more of that would help user confidence. https://kubernetes.io/docs/concepts/workloads/controllers/deployment/

Reply:
Updated.